### ISSUE TITLE
Convert_Missions throws Error when only 1 Mission Exists

### ISSUE THREAD

**Issue by brentspector** [brentspector/urcollectionmanager#13]
[ERROR] IndexError: list index out of range
Traceback (most recent call last):
  File "/var/task/lambda_handler.py", line 5, in lambda_function
    return email.generate_email()
  File "/var/task/urdailyemail/email.py", line 43, in generate_email
    missions = api.convert_missions(
  File "/var/task/urcollectionmanager/api.py", line 94, in convert_missions
    return find_missions(missions)
  File "/var/task/urcollectionmanager/missions.py", line 24, in find_missions
    return [create_mission(row) for row in page.find_all("ul")[2].find_all("li")]

**Comment by brentspector** [comment]
Output of session call:
```
<ul class="list-unstyled m-0">
<li><div class="mission"><div class="media"><img alt="icon" class="align-self-center mr-3" src="https://s.acdn.ur-img.com/img/v2/missions/icons/blackmarket.png" width="46"/><div class="
media-body align-self-center"><h5 class="media-heading">BLACK MARKET - 35 M2 Sansot -&gt; 30M Clintz<span class="float-right"> <img alt="icon" src="https://s.acdn.ur-img.com/img/v2/miss
ions/categories/gdkdo.png" width="14"/></span></h5><small>Sell 35 doubles of M2 Sansot to Kate</small><div class="progress progress-small"><div class="progress-bar" style="width: 0%;">0
/35</div></div></div></div></div></li>
</ul>

```

**Comment by brentspector** [comment]
0a0c614 fixes this issue and is included in release v1.1.3
